x-pack/plugins/timelines/server/search_strategy/index_fields/index.ts

```diff
@@ -337,7 +311,5 @@ export const formatIndexFields = async (
   responsesFieldSpec: FieldSpec[][],
   indexesAlias: string[]
 ): Promise<IndexField[]> => {
-  const fields = await formatFirstFields(beatFields, responsesFieldSpec, indexesAlias);
-  const secondFields = await formatSecondFields(fields);
-  return secondFields;
+  return formatFirstFields(beatFields, responsesFieldSpec, indexesAlias);
```

Comment:
I think you can get rid of this and just call `formatFirstFields` directly where needed. Only seems to be used in this file right now anyways.

Reply:
> > * I don't think this is under the platform codeowners, but should it be under both? Would you consider it a part of sourcerer overall? If not, happy to leave it to @michaelolo24  :)
> 
> @stephmilovic, could you please include the codeowners file update to this PR?

I actually don't see this directory referenced in CODEOWNERS. Looking at the PR, it only pinged @elastic/security-threat-hunting-investigations. I think @yctercero  might be confusing the ping from [my other PR](https://github.com/elastic/kibana/pull/130917) which is still awaiting platform review

Reply:
Testing that other one now! I'd just been going through the both of your open PRs. It seems like this deals with fetching the data views/fields, so I wondered if it was part of the larger data views architecture. If it's purely timeline and not an area of code platform needs to worry about, then all good 👍🏽